src/frontend/js/widgets/CourseProductItem/components/PurchaseButton/index.tsx

```diff
@@ -56,11 +63,15 @@ const PurchaseButton = ({ product, disabled }: PurchaseButtonProps) => {
     );
   }
 
+  const hasAtLeastOneRemainingOrder =
+    product.remaining_order_count && product.remaining_order_count > 0;
+  const isPurchasable = hasAtLeastOneRemainingOrder && hasAtLeastOneCourseRun;
+
   return (
     <>
       {!disabled && (
         <>
-          <button
+          <Button
```

Comment:
Isn't there visual issue to use Cunningham button on the course syllabus as other course run enrollment button are old ones ?

Reply:
I thought that we want to migrate all buttons to cunningham buttons, so i did this one on the way.
Should i reverse my change ?

Reply:
Yup, we will make this change in a dedicated PR as we have several things to check.